cibyl/models/openstack/node.py

```diff
@@ -27,4 +28,45 @@ class Node(Model):
     """
     # To be implemented in future PR
 
-    API = {}
+    API = {
+        'name': {
+            'attr_type': str,
+            'arguments': [Argument(name='--name', arg_type=str,
+                          description="Node name")]
+        },
+        'node_id': {
+            'attr_type': str,
+            'arguments': [Argument(name='--id', arg_type=str,
+                          description="Id provided for the node")]
+        },
+        'status': {
+            'attr_type': str,
+            'arguments': [Argument(name='--status', arg_type=str,
+                          description="Active or inactive status on node")]
+        },
+        'time_created': {
```

Comment:
Probably can be removed. In most cases we will not have this information and even if we do, I don't think anyone would query for that with --time-created.